source/common/common/BUILD

```diff
@@ -262,11 +279,21 @@ envoy_cc_library(
 envoy_cc_library(
     name = "version_lib",
     srcs = ["version.cc"],
+    hdrs = select({
```

Comment:
This should be in `srcs` unless you are exporting.

Reply:
The reason I had to put them in hdrs is because as far as I could tell (empirically and from [docs](https://docs.bazel.build/versions/master/be/c-cpp.html#hdrs)) is that `strip_include_prefix` only words on `hrds` and not on header files in `srcs`. And I needed to do that in order to be able to have the same filename for the header file in version.cc